src/common-helpers.ts

```diff
@@ -94,6 +94,18 @@ export const getExamples = <
   validate?: boolean;
 }): ReadonlyArray<V extends "parsed" ? z.output<T> : z.input<T>> => {
   const examples = schema._def[metaSymbol]?.examples || [];
+  if (schema instanceof z.ZodObject && !examples.length) {
+    const fromProps = Object.entries(schema.shape as z.ZodRawShape).reduce(
```

Comment:
perhaps it should somehow go to `extractObjectSchema` to support more cases

Reply:
not really, because it should also be applicable to response, which can be anything besides object